green_cli/green.py

```diff
@@ -40,6 +40,8 @@ def _get_authenticator(options):
 @click.option('--auth-config', default=None, help='Additional json config passed to the authenticator')
 @click.option('--config-dir', '-C', default=None, help='Override config directory.')
 @click.option('--compact', '-c', is_flag=True, help='Compact json output (no pretty printing)')
+@click.option('--electrum-url', default=None, type=str, help='Use the given Electrum server (single-sig only)')
+@click.option('--electrum-tls', default=None, type=bool, help='Connect to electrum using TLS (single-sig only)')
```

Comment:
nit: actually this params are used for multi-sig SPV as well

Reply:
IIRC these params should be passed at `connect`, thus should be added to `session_params`

Reply:
I've updated the text description (forgot about SPV).

The params are already added to `session_params` via `optional_keys`.